### PR TITLE
DLS-3215, DLS-3212, DLS-3211, DLS-3210

### PR DESCRIPTION
**Pull request by gavinharris43**
Combination of the following DLS-3215, DLS-3212, DLS-3211, DLS-3210

### CODE REVIEW

File: conf/services/agents.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+serviceName: Agents Online Self-Serve
+serviceHeaderName: Agent services
+serviceDescription: |
+  Agents Online Self-Serve provides a client tax dashboard where agents can view the PAYE liabilities and payments for
+  their clients, amend client details online, access client services such as online filing and payment plans, confirm
+  that they still act for a given client, and access old portal PAYE for agents features such as Tax codes.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /agents
```
Comment (calinursaciuc) [inline conf/services/agents.yml:8]: The service url is /agent if you look in prod routes in agent-frontend

Comment (calinursaciuc) [reply conf/services/agents.yml:8]: Also maybe rename the yml to be agent.yml

File: conf/services/country-by-country-reporting.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+serviceDescription: |
+  There is a statutory requirement for UK headed Multinational enterprises (MNEs), or UK sub-groups of MNEs, to make an
+  annual country-by-country report to provide HMRC with information about global activities, profits and taxes.
+  The country-by-country reporting service provides a digital platform where these entities can file their reports to
+  HMRC.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /country-by-country-reporting
+contactFrontendServiceId: country-by-country-reporting
+complianceStatus: noncompliant
+statementVisibility: draft
```
Comment (calinursaciuc) [inline conf/services/country-by-country-reporting.yml:12]: Shall we make public?

Comment (gavinharris43) [reply conf/services/country-by-country-reporting.yml]: done

File: conf/services/agents.yml (hunk 1 above)
Comment (gavinharris43) [reply conf/services/agents.yml]: done

**Review by calinursaciuc — APPROVED** [review]

**Pull request merged**